addons/medical/CfgWeapons.hpp

```diff
@@ -0,0 +1,17 @@
+class CfgWeapons {
+    class CBA_MiscItem_ItemInfo;
+    class ACE_ItemCore;
+
+    class GVAR(painkillers): ACE_ItemCore {
+        scope = 2;
+        author = "Alganthe";
+        model = "\A3\Structures_F_EPA\Items\Medical\PainKillers_F.p3d";
+        picture = QPATHTOF(data\painkillers_inventory.paa);
```

Comment:
Should be in `ui\painkillers_inventory_ca.paa` (or `_co` if it has no transparency).